vhost_user_backend/src/lib.rs

```diff
@@ -622,5 +632,11 @@ impl<S: VhostUserBackend> VhostUserSlaveReqHandler for VhostUserHandler<S> {
 }
 
 impl<S: VhostUserBackend> Drop for VhostUserHandler<S> {
-    fn drop(&mut self) {}
+    fn drop(&mut self) {
+        if let Some(thread) = self.backend.write().unwrap().vring_worker_thread() {
+            if let Err(e) = thread.join() {
+                error!("Error in vring worker: {:?}", e);
+            }
+        }
+    }
```

Comment:
If backends are in charge of managing the worker threads, why should we join them in VhostUserHandler's Drop? Also, this only deals with a single worker thread.

Reply:
Right, remove this part.